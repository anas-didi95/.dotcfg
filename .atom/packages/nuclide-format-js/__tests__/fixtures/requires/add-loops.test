for (one in two) {}
for (three of four) {}
while (five) {}
for (six; seven; eight) {}
do {} while (nine);
