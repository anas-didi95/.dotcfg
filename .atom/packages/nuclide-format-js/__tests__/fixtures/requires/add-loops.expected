const eight = require('eight');
const five = require('five');
const four = require('four');
const nine = require('nine');
const one = require('one');
const seven = require('seven');
const six = require('six');
const three = require('three');
const two = require('two');

for (one in two) {}
for (three of four) {}
while (five) {}
for (six; seven; eight) {}
do {} while (nine);
